src/components/SortDropdown.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+}));
+
+function SortDropdown(props) {
+  const { items, updateSorting } = props;
+  const label = 'Sorting';
+  const [choice, setChoice] = useState('deadlineSoon');
+  const classes = useStyles();
+
+  function selectChoice(event) {
+    console.log(`switch Value changed to: ${event.target.value}`);
+    setChoice(event.target.value);
+    updateSorting(event.target.value);
+  }
+
+  return (
+    <>
```

Comment:
These aren't needed when there's a single child

Reply:
Ok must of not removed when I used them for including a `Button` onClick() when testing the change of state variables. I'll remove it